src/site/stages/build/process-cms-exports/schemas/raw/paragraph-wysiwyg.js

```diff
@@ -0,0 +1,12 @@
+const { getFilter } = require('../../filters');
+
+/* eslint-disable camelcase */
+module.exports = {
+  type: 'object',
+  properties: {
+    field_wysiwyg: {
+      $ref: 'GenericNestedString',
```

Comment:
The export also includes a `format` key which is set to `rich_text`, but this doesn't appear to be used so I don't think we need to enforce its presence.  We're interested in the `value`.